core/src/test/scala/org/apache/spark/scheduler/SparkListenerSuite.scala

```diff
@@ -476,3 +480,19 @@ private class ListenerThatAcceptsSparkConf(conf: SparkConf) extends SparkListene
   var count = 0
   override def onJobEnd(job: SparkListenerJobEnd): Unit = count += 1
 }
+
+private class SlowShuffleManager(conf: SparkConf) extends SortShuffleManager(conf) {
+  
+  override val shuffleBlockResolver: IndexShuffleBlockResolver =
+    new SlowIndexShuffleBlockResolver(conf)
+    
+    private class SlowIndexShuffleBlockResolver(conf: SparkConf,
```

Comment:
(Can you just make this an anonymous inner class? might be shorter. I might also be missing something.)